features/writingsuggestions.yml

```diff
@@ -0,0 +1,6 @@
+name: 'writingsuggestions'
+description: 'The `writingsuggestions` HTML attribute allows you to turn browser-provided writing suggestions on or off for editable regions of a web page.'
+spec: https://html.spec.whatwg.org/multipage/interaction.html#writing-suggestions
+group: forms
+compat_features:
+  - html.global_attributes.writingsuggestions
```

Comment:
The spec makes it seem like there should be a DOM attribute too. Is that being contributed to BCD as well, or is it merely unimplemented? It's not in the linked PR, in any case.

Reply:
The corresponding DOM property had been added to BCD before my PR: https://github.com/mdn/browser-compat-data/blob/main/api/HTMLElement.json#L2406

Do you think we need to list here too? I can add it for completeness.

Reply:
Yes, please include it. I can't imagine it would belong with any other feature.